packages/opentelemetry-scope-base/package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "@opentelemetry/context-base",
+  "name": "@opentelemetry/scope-base",
   "version": "0.0.1",
-  "description": "OpenTelemetry ContextManager API",
+  "description": "OpenTelemetry Base Scope Management",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.ts",
-    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p ./tsconfig.json test/**/*.ts",
```

Comment:
I think `test`, `tdd` and `codecov` scripts is not required for this package (atleast for now), unless we are adding tests for `BaseScopeManager`.

Reply:
I don't think there is any harm to keep them since the implementation of the base should be coming right after this PR.